mariadb/templates/configmaps/seed.sh.yaml.j2

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  seed.sh: |+
+    #!/bin/sh
+
+    set -ex
+
+    SLEEP_TIMEOUT=5
+    # we need to count seed instance here.
+    MINIMUM_CLUSTER_SIZE=4
```

Comment:
Minimum sensible cluster size is always 3 (4 in this case cause the seed instance is also included). When 3-instance cluster is built, then there's no need to keep seed instance running.